pkg/pillar/cmd/zedrouter/pbr.go

```diff
@@ -253,15 +253,11 @@ func PbrAddrChange(deviceNetworkStatus *types.DeviceNetworkStatus,
 		changed = devicenetwork.IfindexToAddrsAdd(log, change.LinkIndex,
```

Comment:
This whole section of code is now a no-op. Perhaps even the function is a no-op snd can be removed?

Reply:
Yes. I think this function can be removed. What this function does is already done by AddrChange function called from NIM context.